src/comp/comp.py

```diff
@@ -23,49 +23,54 @@ def __repr__(self):
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
+
```

Comment:
Comprehensions all look good Michael. You could omit the empty list initializations but it's very readable as it 👍